modules/site-health/audit-enqueued-assets/helper.php

```diff
@@ -78,14 +78,29 @@ function perflab_aea_get_total_size_bytes_enqueued_styles() {
 
 /**
  * Convert full URL paths to absolute paths.
+ * Covers Standard WP configuration, wp-content outside WP directories and subdirectories.
+ * Ex: https://example.com/content/themes/, https://example.com/wp/wp-includes/
  *
  * @since 1.0.0
  *
  * @param string $resource_url URl resource link.
  * @return string Returns absolute path to the resource.
  */
 function perflab_aea_get_path_from_resource_url( $resource_url ) {
```

Comment:
Im noticing quite a few varying function namespaces - maybe this function should be called `perflab_get_path_from_resource_url`?

Reply:
hello, @dainemawer all functions are prepending "perflab_aea", what are the function namespaces varying? maybe I'm missing some

Reply:
I think the approach of having module specific namespaces (all of them starting with `perflab_` though due to the overall plugin) is a solid approach, which avoids us running into accidental conflicts between individual modules.

Reply:
@manuelRod I was thinking more globally across the plugin - I've noticed other functions are all prefixed with `perflab_` but not necessarily `perflab_aea` - that being said it sounds like @felixarntz is good with this approach - Im not swayed either way - just wanted to call it out for consistency reasons :)